app/assets/stylesheets/modules/_search-filters.scss

```diff
@@ -40,7 +40,7 @@
   border: 1px solid #CAC9C9 !important;  /*Removes border*/
   -webkit-appearance: none;  /*Removes default chrome and safari style*/
   -moz-appearance: none; /* Removes Default Firefox style*/
-  background: #fff url(dropdown-arrow.png) no-repeat right 12px center;
+  background: #fff image-url('dropdown-arrow') no-repeat right 12px center;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Prefer double-quoted strings